kernel/src/parquet.rs

```diff
@@ -0,0 +1,11 @@
+//! This module exists to help re-export the version of arrow used by default-gengine and other
+//! parts of kernel that need arrow
+
+#[cfg(all(feature = "arrow_53", feature = "arrow_54"))]
```

Comment:
these should be the parquet flags:
```suggestion
#[cfg(all(feature = "parquet_53", feature = "parquet_54"))]
```

same below.

But that said, do you think we _ever_ want say `arrow_53` and `parquet_54`? I thought they always co-versioned. Could we just have the arrow feature flags and pull the appropriate parquet crate?

Reply:
Good point... would it even be safe to use arrow-54 and parquet-53 or vice-versa? Seems like we'd quickly run into ABI compat issues?

Reply:
It is definitely not safe to use differing versions of parquet and arrow crates together, we'd run into the same symbol conflicts that led to this work in the first place :laughing: